src/Tribe/Hooks.php

```diff
@@ -4,73 +4,122 @@
 	 *
-	 * @since __TRIBE_VERSION__
+	 * @since 1.0.0
 	 */
 	public function register() {
 		$this->container->singleton( static::class, $this );
-		$this->container->singleton( 'extension.__TRIBE_SLUG_CLEAN__.hooks', $this );
+		$this->container->singleton( 'extension.promoter_wp_admin_integration.hooks', $this );
 
 		$this->add_actions();
 		$this->add_filters();
 	}
 
 	/**
 	 * Adds the actions required by the plugin.
-	 *
-	 * @since __TRIBE_VERSION__
+	 * 
+	 * @since 1.0.0
 	 */
 	protected function add_actions() {
 		add_action( 'tribe_load_text_domains', [ $this, 'load_text_domains' ] );
 	}
 
 	/**
 	 * Adds the filters required by the plugin.
-	 *
-	 * @since __TRIBE_VERSION__
+	 * 
+	 * @since 1.0.0
 	 */
 	protected function add_filters() {
-
+		/** @var Tribe__Promoter__PUE $pue */
+		$pue = tribe( 'promoter.pue' );
+		if ( ! $pue->has_license_key() ) {
+			return;
+		}
+		add_filter( 'post_row_actions', [ $this, 'filter_actions' ], 30, 2 );
+		add_filter( 'tribe_tickets_attendees_event_action_links', [ $this, 'add_promoter_action_link' ], 10, 2 );
 	}
 
 	/**
 	 * Load text domain for localization of the plugin.
-	 *
-	 * @since __TRIBE_VERSION__
+	 * 
+	 * @since 1.0.0
 	 */
 	public function load_text_domains() {
 		$mopath = tribe( Plugin::class )->plugin_dir . 'lang/';
-		$domain = '__TRIBE_SLUG__';
+		$domain = 'tribe-ext-promoter-wp-admin-integration';
 
 		// This will load `wp-content/languages/plugins` files first.
 		\Tribe__Main::instance()->load_text_domain( $domain, $mopath );
 	}
+
+	/**
+	 * Add Promoter links to the post list table for tribe_events
+	 * 
+	 * @param array    $actions
+	 * @param WP_Post  $post
+	 * 
+	 * @since 1.0.0
+	 */
+	public function filter_actions( $actions, $post ) {
+		// Only proceed if we're viewing a tribe_events post type.
+		if ( !$post->post_type == 'tribe_events' ) {
+			return $actions;
+		}
+
+		// Only proceed if there are tickets.
+		if ( !tribe_events_has_tickets( $post ) ) {
+			return $actions;
+		}
+
+		$actions['promoter_'] = sprintf(
+			'<a title="%s" href="%s" target="_blank">%s</a>',
+			esc_html__( 'See this event in Promoter', 'tribe-ext-promoter-wp-admin-integration' ),
+			esc_url( 'https://promoter.theeventscalendar.com/events/' . $post->ID ),
+			esc_html__( 'Promoter', 'tribe-ext-promoter-wp-admin-integration' )
+		);
+
+		return $actions;
+	}
+
+	/**
+	 * Add Promoter action links to the attendees page
+	 * 
+	 * @param array  $action_links
+	 * @param string $event_id
+	 * 
+	 * @since 1.0.0
+	 */
+	public function add_promoter_action_link( $action_links, $event_id ) {
+		$action_links[] = '<a href="' . esc_url( 'https://promoter.theeventscalendar.com/messages/new/' . $event_id ) . '" title="' . esc_attr_x( 'Create Promoter Message', 'attendee event actions', 'tribe-ext-promoter-wp-admin-integration' ) . '">' . esc_attr_x( 'Create Promoter Message', 'attendee event actions', 'tribe-ext-promoter-wp-admin-integration' ) . '</a>';
```

Comment:
Just for consistency and to make things a bit more consistent you can break this to the same as the other one using a `sprintf` statement.

```suggestion
		$action_links[] = sprintf(
			"<a href='%s' title='%s'>%s</a>",
			esc_url( 'https://promoter.theeventscalendar.com/messages/new/' . $event_id ),
			esc_attr_x( 'Create Promoter Message', 'attendee event actions', 'tribe-ext-promoter-wp-admin-integration' ),
			esc_html_x( 'Create Promoter Message', 'attendee event actions', 'tribe-ext-promoter-wp-admin-integration' )
		);
```